pkgs/tools/package-management/dpkg/default.nix

```diff
@@ -4,11 +4,11 @@
 
 stdenv.mkDerivation rec {
   pname = "dpkg";
-  version = "1.21.1ubuntu2.1";
+  version = "1.21.9ubuntu1";
```

Comment:
Since the time this PR was opened dpkg 1.21.17 was released.

https://salsa.debian.org/dpkg-team/dpkg/-/blob/1.21.17/debian/changelog